core/src/main/scala/org/http4s/EntityDecoder.scala

```diff
@@ -72,70 +75,98 @@ object EntityDecoder extends EntityDecoderInstances {
   private val UndefinedMediaType = new MediaType("UNKNOWN", "UNKNOWN")
 
   /** summon an implicit [[EntityEncoder]] */
-  def apply[F[_], T](implicit ev: EntityDecoder[F, T]): EntityDecoder[F, T] = ev
-
-  /** Create a new [[EntityDecoder]]
-    *
-    * The new [[EntityDecoder]] will attempt to decode messages of type `T`
-    * only if the [[Message]] satisfies the provided [[MediaRange]]s
-    */
-  def decodeBy[F[_]: Applicative, T](r1: MediaRange, rs: MediaRange*)(
-      f: Message[F] => DecodeResult[F, T]): EntityDecoder[F, T] = new EntityDecoder[F, T] {
-    override def decode(msg: Message[F], strict: Boolean): DecodeResult[F, T] =
-      try {
-        if (strict) {
-          msg.headers.get(`Content-Type`) match {
-            case Some(c) if matchesMediaType(c.mediaType) => f(msg)
-            case Some(c) => DecodeResult.failure(MediaTypeMismatch(c.mediaType, consumes))
-            case None if matchesMediaType(UndefinedMediaType) => f(msg)
-            case None => DecodeResult.failure(MediaTypeMissing(consumes))
-          }
-        } else {
-          f(msg)
+  def apply[M[_[_]], F[_], T](implicit ev: EntityDecoder[F, T]): EntityDecoder[F, T] = ev
```

Comment:
Where is `M` used here?

Reply:
It's not thats a leftover from when I had the `EntityDecoder` parameterized on `M`